setup.py

```diff
@@ -550,6 +551,30 @@ def setupButtonEvents(action, channel, NoC, VoV, msgType, deviceID):
             selectedFilter = printArraySelect(tempFilterList)
         action = jsonArchive["ToggleSourceFilter"] % (source, selectedFilter, "%s")
         saveTODOButtonToFile(channel, msgType, NoC, VoV, "button" , action, "ToggleSourceFilter", source, selectedFilter, deviceID)
+    elif action == "SetAudioMonitor":
+        updateSceneList()
+        updateSpecialSources()
+
+        tempSceneList = []
+        for scene in sceneListLong:
+            for line in scene["sources"]:
+                if line["name"] not in tempSceneList:
+                    tempSceneList.append(line["name"])
+        for item in specialSourcesList:
+            tempSceneList.append(item)
+        source = printArraySelect(tempSceneList)
+        tempArray = ["0 (None)", "1 (Monitor Only)", "2 (Monitor and Output)"]
```

Comment:
You can get rid of the numbers in front of the items and following that the backets. The printArraySelect function will add the numbers for you automatically.

Reply:
That should be it :) Glad I could help, thanks!